src/crawlers/cheerio_crawler.js

```diff
@@ -339,7 +331,7 @@ class CheerioCrawler {
             'CheerioCrawler: requestFunction timed out.',
         );
 
-        request.loadedUrl = response.request.uri.href;
+        request.loadedUrl = response.request.gotOptions.href;
```

Comment:
Are we sure this is the final URL after all redirects? I would expect `gotOptions` to include the input URL.